tests/run-fvttests.sh

```diff
@@ -4,16 +4,90 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-##############################################################################################
-# Export the following Environment variables so that they are accessible to the child script 
-# run-engine-taurus-deploy.sh invoked from this script.                   
-##############################################################################################
-export TRUSTSTORE_PASSWORD="${FHIR_TRUSTSTORE_PASSWD}"
-export TRUSTSTORE_TYPE="pkcs12"
-export TRUSTSTORE="tests\/src\/main\/resources\/config\/trustStore.pkcs12"
-export FHIR_USER_PASS="${FHIR_USER_PASSWORD}"
-
-cd tests
-
-# call the run-taurus-deploy.sh script passing in the final xml output file name (-o option) and a comma separated list of test yaml files (-t file1, file2 ..)
-bash ./run-engine-taurus-deploy.sh -o fvttest.xml -t tests/scenarios/breastCancerTestScenarios.yaml,tests/scenarios/coloRectalCancerTestScenarios.yaml,tests/scenarios/lungCancerTestScenarios.yaml,tests/scenarios/measureCLIExample-separate-measure-server.yaml
+#!/bin/bash
+
+# Helper to run bzt on the pod specifying a test yaml file to be executed
+runTest() {
+  echo "Running test" ${1}
+  kubectl -n ${CLUSTER_NAMESPACE} exec ${POD_NAME} -- bash -c "export TRUSTSTORE_PASSWORD=${TRUSTSTORE_PASSWORD} && bzt ${1}"
+}
+
+# Replace jar, server details, and filename in scenarios yaml files
+populateTaurusYaml() {
+  yamlfile=$1
+  xmlfile=$2
+  
+  sed -i "/JAR/s|:.*$|: \"${SHADED_JAR}\"|" ${yamlfile}
+  sed -i "/DATA_FHIR_SERVER_DETAILS/s|:.*$|: \"${TESTFVT_TENANT}\"|" ${yamlfile}
+  sed -i "/TERM_FHIR_SERVER_DETAILS/s|:.*$|: \"${TESTFVT_TENANT}\"|" ${yamlfile}
+  sed -i "/filename/s|:.*$|: \"${xmlfile}\"|" ${yamlfile}
+}
+
+. tests/setupEnvironmentVariables.sh
+
+POD_NAME=engine-cohort-fvt-test
+
+bash tests/create-taurus-pod.sh ${POD_NAME}
+
+BREAST_XMLFILE="/bzt-configs/tests/results/breastCancerTestScenarios.xml"
+COLORECTAL_XMLFILE="/bzt-configs/tests/results/coloRectalCancerTestScenarios.xml"
+LUNG_XMLFILE="/bzt-configs/tests/results/lungCancerTestScenarios.xml"
+MEASURECLI_XMLFILE="/bzt-configs/tests/results/measureCLITests.xml"
+
+
+# Update yaml scenarios specific to FVT testing
+# breastCancerTestScenarios
+populateTaurusYaml ${SCENARIOS_DIR}/breastCancerTestScenarios.yaml ${BREAST_XMLFILE}
+
+# coloRectalCancerTestScenarios
+populateTaurusYaml ${SCENARIOS_DIR}/coloRectalCancerTestScenarios.yaml ${COLORECTAL_XMLFILE}
+
+# lungCancerTestScenarios
+populateTaurusYaml ${SCENARIOS_DIR}/lungCancerTestScenarios.yaml ${LUNG_XMLFILE}
+
+# measureCLIExample-separate-measure-server.yaml and accompanying json file
+populateTaurusYaml ${SCENARIOS_DIR}/measureCLIExample-separate-measure-server.yaml ${MEASURECLI_XMLFILE}
+sed -i "s|\"cohort-cli/config/local-ibm-fhir.json\"|\"${DEFAULT_TENANT}\"|g" ${TEST_DIR}/src/main/resources/measureCLIExample-separate-measure-server.json
+
+# Copy scenarios into the pod
```

Comment:
Not sure why you need to copy scenarios into the pod since the create-taurus-pod.sh already copies everything under the tests folder down into /bzt-configs  unless i am missing something.

Reply:
The create script copies everything  under `tests/` as it appears in the repo. These copy statements replace some of the key files with their updated versions that are specific to the FVT tests.